pkg/controller/jobs/leaderworkerset/leaderworkerset_reconciler.go

```diff
@@ -165,6 +166,9 @@ func (r *Reconciler) createPrebuiltWorkload(ctx context.Context, lws *leaderwork
 
 	priorityClassName, source, p, err := jobframework.ExtractPriority(ctx, r.client, lws, createdWorkload.Spec.PodSets, nil)
 	if err != nil {
+		if apierrors.IsNotFound(err) {
```

Comment:
How can we determine whether it’s a WorkloadPriorityClass not found or a PriorityClass not found?